### PR TITLE
Pype global settings

### PR DESCRIPTION
**Pull request by iLLiCiTiT**
## Changes
- added new type of settings named `global_settings`
- `global_settings` are saved on each system settings save
    - the values of global settings can't be modified directly
- changed how global settings are loaded in igniter
    - `pype_path` should be loading as expected
- removed not used `load_environments` function from `igniter.tools`

### CODE REVIEW

**Comment by mkolar** [comment]
We need to store it unders it key, otherwise we won't be able to add anything else if need be in the future
image

**Comment by iLLiCiTiT** [comment]
> We need to store it unders it key,

Well spotted. Fixed.

**Review by mkolar — APPROVED** [review]

File: igniter/tools.py [hunk 1]
```diff
@@ -185,73 +187,66 @@ def validate_path_string(path: str) -> (bool, str):
...
-def get_pype_path_from_db(url: str) -> Union[str, None]:
-    """Get Pype path from database.
+def get_pype_global_settings(url: str) -> dict:
+    """Load global settings from Mongo database.
 
     We are loading data from database `pype` and collection `settings`.
     There we expect document type `global_settings`.
 
-    Args:
-        url (str): mongodb url.
```
Comment (antirotor) [inline igniter/tools.py:227]: shouldn't `url` argument be documented?

File: igniter/tools.py [hunk 2]
```diff
@@ -185,73 +187,66 @@ def validate_path_string(path: str) -> (bool, str):
...
+    """Get Pype path from global settings.
+
+    Args:
+        url (str): mongodb url.
+
+    Returns:
+        path to Pype or None if not found
+    """
+    global_settings = get_pype_global_settings(url)
     return global_settings.get("pype_path", {}).get(platform.system().lower())
```
Comment (antirotor) [inline igniter/tools.py:238]: rest of the code won't execute then?

**Pull request merged**